lib/install/config/webpacker.yml

```diff
@@ -5,6 +5,7 @@ default: &default
   source_entry_path: packs
   public_output_path: packs
   cache_path: tmp/cache/webpacker
+  custom_compile: false
```

Comment:
@dhh @gauravtiwari I need to add something like this so that assets:precompile does not try to use the bin/webpacker command.

We can't just turn compile off, as that's used to determine if compile checking should happen. Maybe instead of true/false, we could have a third value of "custom"? Whatever you prefer is fine with me.